nixos/modules/services/monitoring/prometheus/exporters/ipmi.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  logPrefix = "services.prometheus.exporter.ipmi";
+  cfg = config.services.prometheus.exporters.ipmi;
+in {
+  port = 9290;
```

Comment:
Shouldn't this be an configurable option?

Reply:
It's handled in the parent module [here](https://github.com/NixOS/nixpkgs/blob/1f709c16104f43d0e67a14a5243ae94f1e03bcaf/nixos/modules/services/monitoring/prometheus/exporters.nix#L11).